docs/docs-content/getting-started/vmware/setup.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: "Set up Palette"
+title: "Set up Palette with VMware"
+description: "Learn how to set up Palette with VMware."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["getting-started", "vmware"]
+---
+
+In this guide, you will learn how to set up Palette for use with your VMware user account. These steps are required in
+order to authenticate Palette and allow it to deploy host clusters.
+
+## Prerequisites
+
+The prerequisite steps to getting started with Palette on VMware are as follows.
+
+- A Palette account with [tenant admin](../../tenant-settings/tenant-settings.md) access.
+- A [VMware vSphere](https://docs.vmware.com/en/VMware-vSphere/index.html) user account with the
+  [required permissions](../../clusters/data-center/vmware/permissions.md).
+
+## Enablement
+
+Palette needs access to your VMware user account in order to create and manage VMware resources.
+
+### Create a Palette API Key
+
+<PartialsComponent category="palette-setup" name="create-tenant-api-key" />
+
+### Create and Upload an SSH Key
+
+<PartialsComponent category="palette-setup" name="generate-ssh-key" />
+
+## Validate
+
+You can verify your Palette API key is added.
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Switch to the **Tenant Admin** scope.
+
+3. Navigate to the left **Main Menu** and select **Tenant Settings**.
+
+4. From the **Tenant Settings Menu**, select **API Keys**.
+
+5. Verify the API key is listed in the table with the correct user name and expiration date.
+
+## Next Steps
+
+Now that you set up Palette for use with VMware vSphere, you can start deploying Kubernetes clusters to your VMware user
+account.
```

Comment:
This is not entirely true :( You can only deploy clusters to vmware if you have an active PCG. When you deploy a PCG, a vmware cloud account is automatically created

Reply:
Excellent point. I have edited this section to reference the PCG instead.